### DISCUSSION TITLE
problem when I want to install Klipper

### DISCUSSION THREAD

**Discussion by Unicorn3D** [Guilouz/Klipper-Flsun-Speeder-Pad#192] [General]
I have the following problem when I want to install the official build.
<img width="825" alt="Bildschirmfoto 2024-10-18 um 12 20 27" src="https://github.com/user-attachments/assets/bfb16e22-0f77-484a-8fe3-b0aaadb2ab3c">

**Comment by Unicorn3D** [comment]
oke found the problem. just enter the command
''pip install setuptools==70.3.0''
 enter it. Then install klipper.

**Reply by Drahon99** [reply]
Thank you so much ! I encountered the same issue, you really helped me

**Comment by dallatorretdu** [comment]
I did solved this by running

```
pip install --upgrade setuptools
pip install -U setuptools[core]
```

just writing it here maybe for future issues
